features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/TreeViewTraversal.kt

```diff
@@ -115,6 +123,35 @@ internal class TreeViewTraversal(
     private fun isHidden(view: View): Boolean =
         view.getTag(R.id.datadog_hidden) == true
 
+    @UiThread
+    private fun updateTouchOverrideAreas(view: View) {
+        val touchPrivacy = view.getTag(R.id.datadog_touch_privacy)
+
+        if (touchPrivacy != null) {
+            val locationOnScreen = IntArray(2)
+
+            // this will always have size >= 2
+            @Suppress("UnsafeThirdPartyFunctionCall")
+            view.getLocationOnScreen(locationOnScreen)
+
+            val x = locationOnScreen[0]
+            val y = locationOnScreen[1]
+            val viewArea = Rect(
+                x - view.paddingLeft,
```

Comment:
> Question

Why we need to remove the padding of view for the touch privacy? I think we can include them since it's also the scope of the view